.github/workflows/test.yml

```
@@ -13,8 +13,9 @@ jobs:
     strategy:
       matrix:
         node-version:
-          - '14.17'
-          - 16
+          - 20.8.1
+          - 20
+          - 21
```

```suggestion
          - 22
```

since v21 is EOL at this point and v22 becomes LTS before the end of october